aiokafka/conn.py

```diff
@@ -350,6 +425,10 @@ async def _do_sasl_handshake(self):
             else:
                 req = auth_klass(payload)
                 resp = await self.send(req)
+                resp = cast(
+                    Union[SaslAuthenticateResponse_v0, SaslAuthenticateResponse_v1],
+                    resp,
+                )
```

Comment:
It's required for mypy (but ok for pyright), problem only with union types (I think problem with https://github.com/python/mypy/labels/topic-join-v-union)